.github/workflows/test.yml

```diff
@@ -1,25 +1,53 @@
+name: Test
+
 on:
   pull_request:
     branches:
       - master
 
 jobs:
-  build:
+  prepack-upload:
     runs-on: ubuntu-18.04
-    name: Tests
+    name: Prepack and Upload
     steps:
       - uses: actions/checkout@v1
-      - name: Setup node
-        uses: actions/setup-node@v1
+      - uses: actions/cache@v1
+        with:
+          path: /home/runner/.cache/yarn/v6
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+      - name: Install dependencies
+        run: yarn
+      - name: Run prepack
+        run: yarn prepack
+      - name: Tarball prepacked dist
+        run: tar czvf bigtest.dist.tgz ./packages/*/dist
+      - uses: actions/upload-artifact@v1
         with:
-          node-version: '12.x'
+          name: bigtest.dist.tgz
+          path: bigtest.dist.tgz
+
+  download-test:
```

Comment:
for the same reason, let's just call this "test" the upload/download portion is an implementation detail, and is explicitly declared in the `needs:` clause.